frontend/src/components/MapTooltip/MapTooltip.tsx

```diff
@@ -57,43 +57,47 @@ export let ProviderPopup = withStore('providerIndex')<ProviderProps>(({ store, p
   let index = store.get('providerIndex')
   let maxIndex = features.length - 1
   let indexMessage = (index + 1) + ' of ' + (maxIndex + 1)
-  return (
-    <MapTooltip coordinates={lngLat}>
-      <table>
-        <tbody>
-          <TableRow name='Name' value={features[index].properties.name} />
-          <TableRow name='NPI' value={features[index].properties.npi} />
-          <TableRow name='Address' value={features[index].properties.address} />
-          <TableRow name='Specialty' value={features[index].properties.specialty} />
-          <TableRow name='Health Center Type' value={features[index].properties.center_type} />
-          <TableRow name='Lat' value={formatCoordinate(lngLat.lat)} />
-          <TableRow name='Lng' value={formatCoordinate(lngLat.lng)} />
-        </tbody>
-      </table>
-      <div className='controls Flex -Center'>
-        <IconButton touchRippleColor='white' onClick={() => _togglePrevProvider(store)}>
-          <KeyboardArrowLeft className='icon-button' />
-        </IconButton>
-        {indexMessage}
-        <IconButton touchRippleColor='white' onClick={() => _toggleNextProvider(store, maxIndex)}>
-          <KeyboardArrowRight className='icon-button' />
-        </IconButton>
-      </div >
-    </MapTooltip >
-  )
-}
-)
+  let { properties } = features[index]
+  return <MapTooltip coordinates={lngLat}>
+    <table>
+      <tbody>
+        {properties && <React.Fragment>
+          <TableRow name='Name' value={properties.name} />
```

Comment:
Is it possible to loop on available properties values?

Reply:
No, order isn't stable for objects (same as Python IIRC).

Reply:
:(